tests/test_project.py

```diff
@@ -111,11 +86,7 @@ def test_workspace_directory(self):
         assert os.path.join(self._tmp_pr, "workspace") == self.project.workspace
 
     def test_config_modification(self):
-        # In-memory modification of the project configuration is
-        # deprecated as of 1.3, and will be removed in version 2.0.
-        # This unit test should reflect that change beginning 2.0,
-        # and check that the project configuration is immutable.
-        assert _CURRENT_VERSION < _VERSION_2
```

Comment:
This test was going to fail when we bumped the version to 2.0.0. I fixed it here in advance of the actual version bump.

Reply:
For reference, this was intended to guard against releasing 2.0 with code that was previously removed in #608. This comment lingered on past when the offending code was removed, which is why it's safe to remove now with no additional changes.